internal/blueprint/blueprint_test.go

```diff
@@ -0,0 +1,136 @@
+// Package blueprint contains primitives for representing weldr blueprints and
+// translating them to OSBuild pipelines
+package blueprint
+
+import (
+	"encoding/json"
+	"reflect"
+	"testing"
+)
+
+var ExpectedQcow2Pipeline string = `{
```

Comment:
I think instead of this you could just update the sample test I wrote with the right json string, right?

Reply:
Yes. Once the PR gets merged, I can rebase and replace it.

Reply:
Great. That's been merged now :)